src/plugin-manager.js

```diff
@@ -8,7 +8,7 @@ import {FrameworkConfiguration} from 'aurelia-framework';
  * @export
  * @interface PluginDefinition
  */
-export interface PluginDefinition {
+interface PluginDefinition {
```

Comment:
Why did you remove the export? That instantly makes it a BC break :(

Reply:
@RWOverdijk  this one also :smile: